lib/dns/records.js

```diff
@@ -64,14 +64,72 @@ const types = {
   TEXT: 13, // TXT
   LOCATION: 14, // LOC
   // new record
-  MAGNET: 15, // TXT
+  MAGNET: 15, // URI
   DS: 16, // DS
   TLS: 17, // TLSA
   SMIME: 18, // SMIMEA
   SSH: 19, // SSHFP
   PGP: 20, // OPENPGPKEY
   // new record
-  ADDR: 21 // TXT
+  ADDR: 21 // URI
+};
+
+/**
+ * Record Types By Value
+ * @constant
+ */
+
+const typesByVal = {
+  [types.INET4]: 'INET4',
+  [types.INET6]: 'INET6',
+  [types.ONION]: 'ONION',
+  [types.ONIONNG]: 'ONIONNG',
+  [types.NAME]: 'NAME',
+  [types.GLUE]: 'GLUE',
+  [types.CANONICAL]: 'CANONICAL',
+  [types.DELEGATE]: 'DELEGATE',
+  [types.NS]: 'NS',
+  [types.SERVICE]: 'SERVICE',
+  [types.URI]: 'URI',
+  [types.EMAIL]: 'EMAIL',
+  [types.TEXT]: 'TEXT',
+  [types.LOCATION]: 'LOCATION',
+  [types.MAGNET]: 'MAGNET',
+  [types.DS]: 'DS',
+  [types.TLS]: 'TLS',
+  [types.SMIME]: 'SMIME',
+  [types.SSH]: 'SSH',
+  [types.PGP]: 'PGP',
+  [types.ADDR]: 'ADDR'
+};
+
+/**
+ * DNS RR Types by Handshake
+ * record type.
+ * @constant
+ */
+
```

Comment:
Not sure what to call this data structure. It is used to dynamically map between Handshake record type and DNS record type. Any feedback would be appreciated.

Reply:
@boymanjor suggested `hnsToDns`

Reply:
I haven't been able to find a better name for this data structure and have began to like its current name. It is very descriptive of what it contains.

Reply:
`dnsTypeToType`

Reply:
indicating that values are DNS types is a good idea. `hnsToDnsTypes` or similar might be good.

Reply:
Ended up calling it `dnsByType` and have assertions in the test suite that the values are correct. Every time that we talk about what to call it, we come up with a different name :man_shrugging: